Example/PaymentSheet Example/PaymentSheet Example/SavedPaymentMethodSheetTestPlayground.swift

```diff
@@ -200,60 +222,20 @@ extension SavedPaymentMethodSheetTestPlayground {
             self.customerId = customerId
             StripeAPI.defaultPublishableKey = publishableKey
 
-            DispatchQueue.main.async {
+            Task {
                 let configuration = self.savedPaymentMethodSheetConfiguration(customerId: customerId, ephemeralKey: ephemeralKey)
-                self.savedPaymentMethodsSheet = SavedPaymentMethodsSheet(configuration: configuration, delegate: self)
+                self.savedPaymentMethodsSheet = SavedPaymentMethodsSheet(configuration: configuration, customer: self.customerAdapter!)
```

Comment:
We can get rid of the force unwrap in `self.customerAdapter!` by creating a separate function like:
```
    func customerAdapter(customerId: String, ephemeralKey: String) -> StripeCustomerAdapter? {
        let customerAdapter: StripeCustomerAdapter?
        switch paymentMethodMode {
        case .setupIntent:
            customerAdapter = StripeCustomerAdapter(customerEphemeralKeyProvider: {
                // This should be a block that fetches this from your server
                //.init(customerId: customerId, ephemeralKeySecret: ephemeralKey)
                return await self.someFunction(customerId: customerId, ephemeralKey: ephemeralKey)
            }, setupIntentClientSecretProvider: {
                return try await self.backend.createSetupIntent(customerId: customerId)
            })
        case .createAndAttach:
            customerAdapter = StripeCustomerAdapter(customerEphemeralKeyProvider: {
                // This should be a block that fetches this from your server
                .init(customerId: customerId, ephemeralKeySecret: ephemeralKey)
            })
        }
        return customerAdapter
    }
```
Then....

```
                guard let customerAdapter = self.customerAdapter(customerId: customerId, ephemeralKey:ephemeralKey) else {
                    return
                }
                let configuration = self.savedPaymentMethodSheetConfiguration()
...
```